packages/at_secondary_server/lib/src/utils/handler_util.dart

```diff
@@ -14,6 +14,14 @@ HashMap<String, String?> getVerbParam(String regex, String command) {
   return verbParams;
 }
 
+int validateTTR(int ttrMs) {
```

Comment:
need local impl of validateTTR so we can set ttl from a non-zero value back to a zero value